src/README.md

```diff
@@ -18,9 +18,9 @@
 
 ### Ready to dive in?
 
-Click the link below to learn how to build your first Yew app and learn from community example projects
+Click the link below to learn how to build your first Yew app and learn from community example projects:
 
-{% page-ref page="getting-started/project-setup/" %}
+[Getting Started](../getting-started/project-setup)
```

Comment:
I personally prefer how it looked before rather than just having a regular link.

<img width="825" alt="Screenshot 2020-07-09 at 11 22 34" src="https://user-images.githubusercontent.com/42674621/87028416-7e0a7c80-c1d6-11ea-8b09-df11f363ad47.png">